### ISSUE TITLE
Windows Releases

### ISSUE THREAD

**Issue by tyeeman** [renatoathaydes/magnanimous#2]
They don't have .exe on their files. Please add.

**Comment by renatoathaydes** [comment]
I use Go to generate the binaries. Can you help find out what may be wrong in the release script?

https://github.com/renatoathaydes/magnanimous/blob/master/Makefile#L51

Which is run on Travis CI:

https://github.com/renatoathaydes/magnanimous/blob/master/.travis.yml

**Comment by tyeeman** [comment]
I think you just add .exe like so - 

```
release-windows:
	env GOOS=windows env GOARCH=amd64 go build -ldflags "-s -w" -o releases/magnanimous-windows-amd64.exe
        env GOOS=windows env GOARCH=386 go build -ldflags "-s -w" -o releases/magnanimous-windows-386.exe
```

**Comment by renatoathaydes** [comment]
This should be fixed now :) sorry for the long time it took... I've just released version 0.9.0.
